.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+This is a (multiple allowed):
+* [x] bug
+* [ ] enhancement
+* [ ] feature-discussion (RFC)
+
+* BootstrapUI Version: EXACT RELEASE VERSION (THE VERSION IS IN YOUR COMPOSER.LOCK FILE).
+* Twitter Bootstrap: FRAMEWORK VERSION
+* jQuery: LIBRARY VERSION
+* Platform and Target: YOUR WEB-SERVER, DATABASE AND OTHER RELEVANT INFO AND HOW THE REQUEST IS BEING MADE, HERE.
+
+### What you did
+EXPLAIN WHAT YOU DID, PREFERABLY WITH CODE EXAMPLES, HERE.
+
+### Expected Behavior
+EXPLAIN WHAT IS TO BE EXPECTED, HERE.
+
+### Actual Behavior
+EXPLAIN WHAT IS ACTUALLY HAPPENING, HERE.
+
+Before you open an issue, please check if a similar issue already exists or has been closed before.
+
+P.S. Remember, an issue is not the place to ask questions.
```

Comment:
If we are gonna remind people not to use issues for question then also need to tell them where they can ask questions instead :)